### ISSUE TITLE
iOS - clearAllNotifications does not clear notifications, even though success is reported

### ISSUE THREAD

**Issue by idekterev** [phonegap/phonegap-plugin-push#2227]
### Expected Behaviour
push.clearAllNotifications(success, failure), should clear all notifications (upon success) in iOS.

### Actual Behaviour
push.clearAllNotifications(success, failure), returns 'success' , however, all notifications are still there.

#### Steps to Reproduce 
init cordova project
init PushNotification object
```
let pushService = PushNotification.init({
  "android": {
    "icon": "fcm_push_icon",
    "clearBadge": "true"
  },
  "browser": {
    pushServiceURL: 'http://push.api.phonegap.com/v1/push'
  },
  "ios": {
    "alert": "true",
    "badge": "true",
    "sound": "true",
    "clearBadge": "true"
  },
  "windows": {}
});
```
get token from registration event:
```
pushService.on('registration', function (data) { 
   console.log(data.registrationId);
});
```

send some push notifications.

then try and clear them:
```
pushService.clearAllNotifications(
  function() {	console.log('success') }, 
  function() {	console.log('error') }
);
```
Success handler is called, however, notifications are still there.

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
iOS 11.2.6

#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
Google Pixel XL

#### Cordova CLI version and cordova platform version

    cordova --version                                    # 8.0.0
    cordova platform version android          # 6.3.0

#### Plugin version

    cordova plugin version | grep phonegap-plugin-push   # 2.1.3

#### Sample Push Data Payload
```
{
  "notification":{
    "title":"My Title",
    "body":"send me a few of these",
    "sound": "default",
  },
  "registration_ids": ["my_reg_id"],
}
```

#### Logs taken while reproducing problem
'success' from clearNotifications can be seen in Safari logs.
PushPlugin clearing badge' on the next line.

**Comment by macdonst** [comment]
@idekterev I'm confused, you are saying iOS but your device is an Android. Which platform is running into the problem.

**Comment by idekterev** [comment]
@macdonst yes indeed - you're not the only one confused :) - I swap around between devices so often , clearly I am confused also. Its an iPhone 7.

**Comment by dmatora** [comment]
doesn't work for me either, but in my case it won't even fire any callback
```
      PushNotification.clearAllNotifications(
          function() {	console.log('success') },
          function() {	console.log('error') }
      );
```
being simply ignored
